tests/keras/backend/reference_operations.py

```diff
@@ -352,11 +361,17 @@ def cumprod(x, axis=0):
 
 
 def any(x, axis=None, keepdims=False):
-    return np.any(x, axis=axis, keepdims=keepdims)
+    a = axis
+    if isinstance(a, list):
+        a = tuple(a)
+    return np.any(x, axis=a, keepdims=keepdims)
 
 
 def all(x, axis=None, keepdims=False):
-    return np.all(x, axis=axis, keepdims=keepdims)
+    a = axis
+    if isinstance(a, list):
+        a = tuple(a)
```

Comment:
Would it be possible to simplify by doing 
```python
if axis is not None:
    axis = tuple(axis)
```

Reply:
This would fail when axis is an integer

Reply:
You are right. My bad. I dont think it's necessary to use an intermediate variable though:

```python
if isinstance(axis, list):
    axis = tuple(axis)
```

Reply:
thanks!